lib/Braintree/Gateway.php

```diff
@@ -18,91 +22,163 @@ public function __construct($config)
         $this->config = $config;
     }
 
+    /**
+     * 
+     * @return \Braintree_AddOnGateway
```

Comment:
are the \'s meaningful? I'm guessing so because it's used in a number of the ones below, but didn't see it above